packages/connect-examples/expo-playground/app/routes/chains._index.tsx

```
@@ -69,18 +69,18 @@ const ChainsIndexPage: React.FC = () => {
-            <div className="grid gap-2 grid-cols-1 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 2xl:grid-cols-5 pb-4">
+            <div className="grid gap-3 grid-cols-1 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 2xl:grid-cols-4 pb-4">
               {filteredChains.map(chain => (
                 <div
                   key={chain.id}
-                  className="flex items-center gap-3 p-3 bg-card border border-border rounded-lg cursor-pointer hover:border-primary/50 hover:bg-primary/5 transition-all duration-200"
+                  className="flex items-center gap-3 p-4 bg-card border border-border rounded-lg cursor-pointer hover:border-primary/50 hover:bg-primary/5 transition-all duration-200 hover:shadow-sm"
                   onClick={() => handleChainSelect(chain)}
                   onKeyDown={e => handleKeyDown(e, () => handleChainSelect(chain))}
                   tabIndex={0}
                   role="button"
                   aria-label={t('chains.exploreChain', { chainId: chain.id })}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)